static_src/components/users_invite.jsx

```diff
@@ -0,0 +1,73 @@
+
+/**
+ * Renders a form that allows org users to invite new users
+ * to cloud.gov
+ */
+
+import React from 'react';
+
+import { Form, FormText } from './form';
```

Comment:
It's hard to enforce unfortunately, because I don't think our linter supports it. It's also not a perfect rule because of the `{ }` es6 syntax, but should at least be kinda organized if that makes sense?